hrm-service/src/contact-job/client-sqs.ts

```diff
@@ -1,11 +1,56 @@
-export const pollCompletedContactJobsFromQueue = async (): Promise<{
-  Messages: { ReceiptHandle: string; Body: string }[];
-}> => {
-  return {
-    Messages: [],
-  };
+import { SQS } from 'aws-sdk';
+import { getSsmParameter } from '../config/ssmCache';
+
+// eslint-disable-next-line prettier/prettier
+import type { PublishToContactJobsTopicParams } from '@tech-matters/hrm-types/ContactJob';
+
+let sqs: SQS;
+
+export const getSqsClient = () => {
+  if (!sqs) {
+    sqs = new SQS();
+  }
+  return sqs;
 };
 
+export const pollCompletedContactJobsFromQueue =
+  async (): Promise<SQS.Types.ReceiveMessageResult> => {
+    try {
+      const QueueUrl = getSsmParameter(
+        `/${process.env.NODE_ENV}/sqs/jobs/contact/queue-url-contact-complete`,
+      );
+
+      return await getSqsClient()
+        .receiveMessage({
+          QueueUrl,
+          MaxNumberOfMessages: 10,
+          WaitTimeSeconds: 0,
+        })
+        .promise();
+    } catch (err) {
+      console.error('Error trying to poll messages from SQS queue');
+    }
```

Comment:
This error being swallowed here means it won't cause [processContactJobs](https://github.com/techmatters/hrm/blob/7831156f8a72108dc81aa88010fe4bf2734cf244/hrm-service/src/contact-job/contact-job-processor.ts#L14) to "[fail a sweep](https://github.com/techmatters/hrm/blob/7831156f8a72108dc81aa88010fe4bf2734cf244/hrm-service/src/contact-job/contact-job-processor.ts#L37)". I'm cool with that as long as we remember that polling failure does not interrupts the publishing step that comes after (which I actually think might be better),